grid/recruitment/get-stage-changes/maps/__snapshots__/breezy-hr.test.ts.snap

```diff
@@ -0,0 +1,63 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`recruitment/get-stage-changes/breezy-hr GetStageChanges when specified candidate does exist performs correctly 1`] = `
+Ok {
+  "value": Object {
+    "stages": Array [
+      Object {
+        "createdAt": "2022-12-14T11:23:18.812Z",
+        "current": true,
+        "id": "applied",
+        "name": "Applied",
+      },
+      Object {
+        "createdAt": "2022-12-14T11:21:30.696Z",
+        "current": false,
+        "id": "feedback",
+        "name": "Feedback",
+      },
+      Object {
+        "createdAt": "2022-12-14T11:19:32.247Z",
+        "current": false,
+        "id": "feedback",
+        "name": "Feedback",
+      },
+      Object {
+        "createdAt": "2022-12-14T11:15:50.674Z",
+        "current": false,
+        "id": "feedback",
+        "name": "Feedback",
+      },
+      Object {
+        "createdAt": "2022-12-14T11:15:29.811Z",
+        "current": false,
+        "id": "feedback",
+        "name": "Feedback",
+      },
+    ],
+  },
+}
+`;
+
+exports[`recruitment/get-stage-changes/breezy-hr GetStageChanges when specified candidate does not exist returns error 1`] = `
+Err {
+  "error": "MappedHTTPError: Expected HTTP error
+Properties: {
+  \\"title\\": \\"Unknown error\\",
+  \\"detail\\": {
+    \\"type\\": \\"getStreamError\\",
+    \\"message\\": \\"there was an unknown error getting candidate stream.\\"
+  },
+  \\"code\\": \\"UnknownError\\",
```

Comment:
This is not good that the API returns 500 error in this case. I am afraid we are not able to solve it as they return 500 anytime they get wrong path params.

Reply:
Yeah, no way to show some helpful info here 😞